### PR TITLE
Test/transaction apply

### PR DESCRIPTION
**Pull request by bogdan34**

### CODE REVIEW

File: test/core/util/byteSerialization/block.test.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        const byteBlock: Buffer = block.byteSerialize();
+
+        expect(deserialize(byteBlock)).to.deep.equal(blockModel);
+    });
+
+    it('Blocks array', () => {
+        const blocksCopy = blocks.map(block => new BlockModel(block));
+
+        blocksCopy.forEach((block: any) => {
+            block.transactions = block.transactions.map(trs => serializeAssetTransaction(trs));
```
Comment (axelsheva) [inline test/core/util/byteSerialization/block.test.ts]: `byteSerialize` doesn't serialize transactions?

File: test/core/util/byteSerialization/mock.ts [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+export const senderAddress = BigInt('4995063339468361088');
+
+/* tslint:disable:no-magic-numbers */
+
+export const amount = 10000000000;
+
+export const transactionRegister = new Transaction({
+    id: '85519724a269996537c1c3533f912016c34b782678f96a0d1a35459b453611de',
+    type: TransactionType.REGISTER,
+    createdAt: Math.floor(Date.now() / 1000),
```
Comment (axelsheva) [inline test/core/util/byteSerialization/mock.ts]: It's invalid time, we should use epoch time

File: test/core/util/byteSerialization/block.test.ts (hunk 1 above)
Comment (bogdan34) [reply test/core/util/byteSerialization/block.test.ts]: serializeAssetTransaction serialize only asset

File: test/core/util/byteSerialization/mock.ts (hunk 2 above)
Comment (bogdan34) [reply test/core/util/byteSerialization/mock.ts]: getEpochTime is private

**Pull request merged**